cmd/runtime.go

```diff
@@ -333,7 +333,7 @@ func runPkgCmd(subSystem *core.SubSystem, command string, cmd *cobra.Command, ar
 			return fmt.Errorf(apx.Trans("runtimeCommand.error.stoppingContainer"), err)
 		}
 
-		cmdr.Info.Printfln(apx.Trans("runtimeCommand.info.stoppedContainer"), subSystem.Name)
+		cmdr.Info.Printfln(apx.Trans("runtimeCommand.info.stoppedContainer"))
```

Comment:
I'm confused. Why are you removing the name if you added %s to the string?

Reply:
sorry that wasnt very clear.  i removed it from the started/stopped message, but not the starting/stopping message

Reply:
Ah, I see.